src/Components/Home/HomeEditForm.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+//Forms
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+//Styles
+import Swal from 'sweetalert2';
+import {
+  Button,
+  Typography,
+  Container,
+  TextField,
+  Paper,
+  Stack,
+} from '@mui/material';
+import s from '../../Styles/HomeEditForm.module.css';
+//Components
```

Comment:
the code should be enought comment for itself, if you need to add a comment is because the name of the functions are not clear or you need to refactor the code.